sdk/core/azure-core/src/main/java/com/azure/core/implementation/tracing/TracerProxy.java

```diff
@@ -76,4 +96,38 @@ public static Context setSpanName(String spanName, Context context) {
 
         return local;
     }
+
+    /**
+     * For each tracer plugged into the SDK the current tracing span is marked as completed.
+     *
+     * @param errorCondition the value of error condition
+     * @param throwable Potential throwable that happened during the span.
+     * @param context Additional metadata that is passed through the call stack.
+     */
+    public static void end(String errorCondition, Context context, Throwable throwable) {
+        tracers.forEach(tracer -> tracer.end(errorCondition, context, throwable));
+    }
+
+    /**
+     * For each tracer plugged into the SDK a link is created between the parent tracing span and
+     * the current service call.
+     *
+     * @param context Additional metadata that is passed through the call stack.
+     */
+    public static void addLink(Context context) {
+        tracers.forEach(tracer -> tracer.addLink(context));
+    }
+
+    /**
+     * For each tracer plugged into the SDK a new context is extracted from the event's diagnostic Id.
+     *
+     * @param diagnosticId Unique identifier of an external call from producer to the queue.
+     */
+    public static Context extractContext(String diagnosticId) {
+        Context local = Context.NONE;
+        for (Tracer tracer : tracers) {
+            local = tracer.extractContext(diagnosticId);
```

Comment:
If there are multiple tracers, wouldn't this `local` context be overwritten and only the last value retained?

Reply:
updated to keep adding the context for each tracer to `local` context object.